database/store.py

```diff
@@ -1,8 +1,13 @@
 import datetime
+from tkinter import SE
```

Comment:
This import appears to be unused and causes problems running the app with vanilla Python, can it be removed?